odoo/addons/base/models/ir_fields.py

```diff
@@ -297,16 +298,29 @@ def _str_to_datetime(self, model, field, value):
 
     @api.model
     def _get_translations(self, types, src):
+        if not src:
+            return []
         types = tuple(types)
         # Cache translations so they don't have to be reloaded from scratch on
         # every row of the file
         tnx_cache = self._cr.cache.setdefault(self._name, {})
         if tnx_cache.setdefault(types, {}) and src in tnx_cache[types]:
             return tnx_cache[types][src]
 
-        Translations = self.env['ir.translation']
-        tnx = Translations.search([('type', 'in', types), ('src', '=', src)])
-        result = tnx_cache[types][src] = [t.value for t in tnx if t.value is not False]
+        values = OrderedSet()
+        if 'code' in types:
+            for lang, __ in self.env['res.lang'].get_installed():
+                translations = code_translations.get_python_translations('base', lang)
```

Comment:
Here you only get code translations for module `base`, while the former code retrieved translations from all installed modules.

Reply:
@xmo-odoo Could you help us check if the new code can cover all useful use cases of the old one?
[CRUD] remove ir.translation for ir_fields
https://github.com/odoo/odoo/pull/97692/commits/2b77815e5da4eb40f95550e23f87ec71888e006f